src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphEntityDAOSpec.scala

```diff
@@ -17,15 +15,23 @@ class GraphEntityDAOSpec extends FlatSpec with Matchers with OrientDbTestFixture
   // setup workspace objects
   val wsName = WorkspaceName("myNamespace", "myWorkspace")
 
-  val sample1 = Entity("sample1", "sample",
+  val aliquot1 = Entity("aliquot1", "aliquot", Map("sampleSet" -> AttributeReferenceSingle("sampleSet", "sampleSet3")), wsName)
+  val aliquot2 = Entity("aliquot2", "aliquot", Map.empty, wsName)
+
+  var sample1 = Entity("sample1", "sample",
     Map(
       "type" -> AttributeString("normal"),
       "whatsit" -> AttributeNumber(100),
-      "thingies" -> AttributeValueList(Seq(AttributeString("a"), AttributeBoolean(true)))),
+      "thingies" -> AttributeValueList(Seq(AttributeString("a"), AttributeBoolean(true))),
+      "aliquot" -> AttributeReferenceSingle("aliquot", "aliquot1")),
     wsName)
 
-  val sample2 = Entity("sample2", "sample", Map("type" -> AttributeString("tumor")), wsName)
+  val sample2 = Entity("sample2", "sample", Map("type" -> AttributeString("tumor"), "aliquot2" -> AttributeReferenceSingle("aliquot", "aliquot2")), wsName)
   val sample3 = Entity("sample3", "sample", Map("type" -> AttributeString("tumor")), wsName)
+  val sample4 = Entity("sample4", "sample", Map("type" -> AttributeString("tumor")), wsName)
+  var sample5 = Entity("sample5", "sample", Map("type" -> AttributeString("tumor")), wsName)
+  var sample6 = Entity("sample6", "sample", Map("type" -> AttributeString("tumor")), wsName)
+  var sample7 = Entity("sample7", "sample", Map("type" -> AttributeString("tumor"), "cycle" -> AttributeReferenceSingle("sample", "sample6")), wsName)
```

Comment:
I suppose not. It was like that because I was adding cycles to the entities after they were saved, but I can just create a new copy of the entity and save that to the workspace to add the cycle.